apps/web-giddh/src/app/new-inventory/new-inventory.component.ts

```diff
@@ -0,0 +1,194 @@
+        }
+        this.hideGiddhDatepicker();
+        if (value && value.startDate && value.endDate) {
+            this.selectedDateRange = { startDate: moment(value.startDate), endDate: moment(value.endDate) };
+            this.selectedDateRangeUi = moment(value.startDate).format(GIDDH_NEW_DATE_FORMAT_UI) + " - " + moment(value.endDate).format(GIDDH_NEW_DATE_FORMAT_UI);
+            this.fromDate = moment(value.startDate).format(GIDDH_DATE_FORMAT);
+            this.toDate = moment(value.endDate).format(GIDDH_DATE_FORMAT);
+        }
+    }
+    /* show/hide funcation search input field */
+    public searhcGroup(){
+        this.inputFullWidth = !this.inputFullWidth
+    }
+    public focusOnInput(){
+
+    }
+    public expandDateRang(){
+        this.dateRangFullWidth = !this.dateRangFullWidth
+    }
+
+
+    /* Aside pane toggle fixed class */
+    public toggleBodyClass(): void {
+        if (this.asideMenuState === 'in') {
+            document.querySelector('body').classList.add('fixed');
+        } else {
+            document.querySelector('body').classList.remove('fixed');
+        }
+    }
+
+    /* Aside pane open function */
+    public toggleAsidePane(event?): void {
+        if (event) {
+            event.preventDefault();
+        }
+        this.asideMenuState = this.asideMenuState === 'out' ? 'in' : 'out';
+        this.toggleBodyClass();
+    }
+
+    /* Create group aside pane open function */
+    public createGroupToggleAsidePane(event?): void {
+        if (event) {
+            event.preventDefault();
+        }
+        this.asideMenuState = this.asideMenuState === 'out' ? 'in' : 'out';
+        this.toggleBodyClass();
+    }
+
+    /* Create item aside pane open function */
+    public createItemToggleAsidePane(event?): void {
+        if (event) {
+            event.preventDefault();
+        }
+        this.asideMenuState = this.asideMenuState === 'out' ? 'in' : 'out';
+        this.toggleBodyClass();
+    }
+
+    /* Create unit aside pane open function */
+    public createUnitToggleAsidePane(event?): void {
+        if (event) {
+            event.preventDefault();
+        }
+        this.asideMenuState = this.asideMenuState === 'out' ? 'in' : 'out';
+        this.toggleBodyClass();
+    }
+
+    /* Create combo aside pane open function */
+    public createComboToggleAsidePane(event?): void {
+        if (event) {
+            event.preventDefault();
+        }
+        this.asideMenuState = this.asideMenuState === 'out' ? 'in' : 'out';
+        this.toggleBodyClass();
+    }
+
+    /**
+    * This will toggle the settings popup
+    *
+    * @param {*} [event]
+    * @memberof SettingsComponent
+    */
+    public toggleSettingPane(event?): void {
+        this.toggleBodyClass();
+
+        if (this.isMobileScreen && event && this.asideInventorySidebarMenuState === 'in') {
+            this.asideInventorySidebarMenuState = "out";
+        }
+    }
+
+    constructor(
```

Comment:
constructor should be at the top of all methods and below member variables

Reply:
Done